### DISCUSSION TITLE
Migrating from toObject pre 1.6.7

### DISCUSSION THREAD

**Discussion by noctivityinc** [fabricjs/fabric.js#8447] [Q&A]
Yes, I know.  1.6.7.  But I am trying to move a 1.5.0 app to the current version and noticing that the changes made to toObject in 1.6.7 are breaking everything.

Specifically, when you save a text a `new fabric.Text` element that TEXT is no longer saved!? So now we get a split error when trying to reload the object.

There were clearly breaking changes from 1.6.6 to 1.6.7 but there is no documentation on how to properly migrate a saved JSON object in 1.6.6 to one that will work when saved in 1.6.7

**Comment by ShaMan123** [comment]
Compare the versions under the versions section

**Comment by DrPokst** [comment]
Have you solved this problem?
